api/tests/integration/infrastructure/repositories/organization-repository_test.js

```diff
@@ -1059,5 +1059,35 @@ describe('Integration | Repository | Organization', function () {
       expect(savedOrganizationFeatures.length).to.equal(1);
       expect(savedOrganizationFeatures[0].featureId).to.equal(computeOrganizationLearnerCertificabilityId);
     });
+
+    it('should associate organizations to enabled features', async function () {
+      // given
+      const missionManagementFeatureId = databaseBuilder.factory.buildFeature(
+        ORGANIZATION_FEATURE.MISSIONS_MANAGEMENT,
+      ).id;
+      const superAdminUserId = databaseBuilder.factory.buildUser.withRole().id;
+      await databaseBuilder.commit();
+      const features = {};
+      features[ORGANIZATION_FEATURE.COMPUTE_ORGANIZATION_LEARNER_CERTIFICABILITY.key] = false;
+      features[ORGANIZATION_FEATURE.MISSIONS_MANAGEMENT.key] = true;
+      const organization = new OrganizationForAdmin({
```

Comment:
Comme `organization-repository` est utilisé, il faut dans ce cas utiliser le modèle `Organization`.

```suggestion
      const organization = new Organization({
```

Reply:
La gestion de l'attribution des features en fonction du type d'orga se fait dans OrganizationForAdmin. Il faudrait donc faire des changements importants pour sortir cette gestion ailleurs et utiliser ici une Organization.